src/applications/personalization/profile/tests/e2e/contact-information/modals.cypress.spec.js

```diff
@@ -335,14 +331,10 @@ describe('when editing other profile fields on the same page', () => {
       force: true,
     });
 
-    cy.get('#root_inputPhoneNumber').should('exist');
+    cy.get('[name="root_inputPhoneNumber"]').should('exist');
 
     // Cancel edit should also exist the edit mode with no modal
-    cy.findByRole('button', {
-      name: /Cancel/i,
-    }).click({
-      force: true,
-    });
+    cy.get('va-button[text="Cancel"]').click({ force: true });
```

Comment:
why do we have to force true here? Is that needed?

Reply:
I included it because it was used before. I didn't test if it was still needed.

Reply:
okay cool maybe in another pr we could check that seems like it may not be needed here!